docs/guide/topics/reactions.rst

```diff
@@ -0,0 +1,133 @@
+Because of this, to add 😄 / ``:smile:`` you would do either of the following:
+
+.. code-block:: python3
+
+    # pass the unicode emoji via the hexadecimal value
+    await message.add_reaction("\U0001f604")
+    
+    # or
+
+    # get the unicode emoji via the name (obtained via the unicode database)
+    await message.add_reaction("\N{SMILING FACE WITH OPEN MOUTH AND SMILING EYES}")
+
+This would then result in:
+
+.. image:: /images/guide/topics/reactions/add_standard_emoji.png
+    :scale: 50%
+
+You can find the documentation for adding reactions to a message object at :meth:`~Message.add_reaction`.
+
+Removing a reaction
+~~~~~~~~~~~~~~~~~~~~~
+
+Removing reactions is similar to adding a reaction, but you must also specify a member to remove the reaction from.
+This can be anything that meets the abc :class:`~abc.Snowflake`, like :class:`~discord.Member` or :class:`~discord.Object`.
+This means that you cannot pass simply only an id, and you must instead pass something with an ``id`` attribute.
+
+.. code-block:: python3
+
+    await message.remove_reaction(
+        "<:amogus:819046807370989599>",
+        guild.me
+    )
+
+This removes a custom emoji reaction with the id ``819046807370989599`` added by us (see :attr:`~Guild.me`) from the message:
+
+.. image:: /images/guide/topics/reactions/remove_reaction_before.png
+    :scale: 50%
+
+.. image:: /images/guide/topics/reactions/remove_reaction_after.png
+    :scale: 50%
+
+Removing all reactions of a single emoji from a message is also possible: 
+
+.. code-block:: python3
+
+    await message.clear_reaction("<:amogus:819046807370989599>")
+
+It is also possible to remove all reactions from a message, not just a singular member or emoji:
+
+.. code-block:: python3
+
+    await message.clear_reactions()
+
+If you have a :class:`~Reaction`, you can also call :meth:`.clear() <Reaction.clear>` on that to clear reactions of that given emoji.
+
+.. note:: 
+    
+    To remove a reaction that is not your own, you will need the :attr:`~Permissions.manage_messages` permission.
+
+Getting reactions
+~~~~~~~~~~~~~~~~~~
+
+The :attr:`~Message.reactions` attribute allows you to get a list of the reactions on the message. This returns a :class:`~Reaction`
+per unique emoji on the message.
+
+For example, to get all the users that reacted with a given emoji:
+
+.. code-block:: python3
+
+    # This fetches the first 5 users that reacted with that emoji.
+    async for user in reaction.users(limit = 5):
+        await channel.send(f"{user} reacted with `{reaction.emoji}")
+
+You can also flatten the users into a list.
+
+.. code-block:: python3
+
+    users = await reaction.users().flatten()
+    # users is now a list of User...
+    winner = random.choice(users)
+    await channel.send(f'{winner} has won the raffle.')
+
+Next Steps
+~~~~~~~~~~~
+
+You can find further information about :class:`~Reaction` in its documentation.
+Additionally, :meth:`~Client.wait_for` can pair well with reactions.
```

Comment:
Might wanna link to the `wait_for` guide here? It doesn't exist yet though.

Reply:
i will link it when the guide exists, but currently it does not.

Reply:
The guide now exists and has a reference link, this can be added.